### ISSUE TITLE
Temporary directory not saved until you exit the addon

### ISSUE THREAD

**Issue by GutHib** [emilsvennesson/script.module.inputstreamhelper#421]
Here's a bug that drove me absolutely nuts today... hope it will save others some headache: Specifying a new temporary download directory and hitting OK will not actually configure&save that directory, until you hit another OK button and leave the addon.

It seemed like all my storage devices were too small or broken or poorly formatted or whatever else. Nope: The addon didn't even look at them...

**Comment by horstle** [comment]
Yes, we discussed this somewhere before. In Kodi, settings are only stored after you close the settings page. Not sure what we could do about it.

**Comment by GutHib** [comment]
You could leave a note on the page where you specify the temporary download directory.

Preferably in red. ;)
